runtime/src/main/java/org/corfudb/runtime/view/stream/AbstractContextStreamView.java

```diff
@@ -87,6 +88,25 @@ public synchronized void reset() {
      * {@inheritDoc}
      */
     @Override
+    public synchronized void seek(long globalAddress) {
+        // pop any stream context which has a max address
```

Comment:
This method is public, probably you need to check that (globalAddres>=0) and throw IllegalArgumentException if necessary